eclair-core/src/test/scala/fr/acinq/eclair/channel/RegisterSpec.scala

```diff
@@ -0,0 +1,30 @@
+package fr.acinq.eclair.channel
+
+import fr.acinq.eclair._
+
+import scala.concurrent.duration._
+import akka.actor.{ActorRef, ActorSystem, Props}
+import akka.testkit.TestProbe
+import fr.acinq.bitcoin.ByteVector32
+import fr.acinq.bitcoin.Crypto.PublicKey
+import org.scalatest.funsuite.AnyFunSuite
+import akka.pattern.ask
+import akka.util.Timeout
+
+import scala.concurrent.Await
+
+class RegisterSpec extends AnyFunSuite {
```

Comment:
You should use the same pattern as other test suites in the repo, see `PostRestartHtlcCleanerSpec` for example:

```suggestion
class RegisterSpec extends AnyFunSuite extends TestKitBaseClass with AnyFunSuiteLike with ParallelTestExecution {
```